src/node/rpc/frontend.h

```diff
@@ -552,15 +570,57 @@ namespace ccf
 
       auto rv = jsonrpc::pack(rep.value(), ctx.pack.value());
 
-#ifdef PBFT
-      if (history)
-        history->add_response(reqid, rv);
+      return rv;
 #endif
+    }
 
-      return rv;
+    /** Process a serialised command with the associated RPC context via PBFT
+     *
+     * If an RPC that requires writing to the kv store is processed on a
+     * follower, the serialised RPC is forwarded to the current network leader.
+     *
```

Comment:
Does forwarding as we do it in Raft make sense in the PBFT context? Followers in PBFT execute their own commands and even if they are not the leader they should get a request, send it to the client proxy, the client proxy will broadcast the request to the other followers (replicas) and then should process it themselves.

Reply:
Sorry, the comment is wrong. There's no command forwarding when processing via PBFT.